CRM/Upgrade/Incremental/sql/4.7.9.mysql.tpl

```diff
@@ -14,3 +14,34 @@ UPDATE civicrm_state_province SET name="Wake Island" WHERE name="Wake Ialand";
 UPDATE civicrm_dashboard SET
   {localize field="label"}label = '{ts escape="sql"}CiviCRM Resources{/ts}'{/localize}
 WHERE name = 'getting-started';
+
+-- CRM-16189 Financial account relationship
+SELECT @option_group_id_arel := max(id) from civicrm_option_group where name = 'account_relationship';
+SELECT @option_group_id_arel_wt  := MAX(weight) FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel;
+SELECT @option_group_id_arel_val := MAX(CAST( `value` AS UNSIGNED )) FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel;
+
+INSERT INTO
+   `civicrm_option_value` (`option_group_id`, {localize field='label'}label{/localize}, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, {localize field='description'}`description`{/localize}, `is_optgroup`, `is_reserved`, `is_active`, `component_id`, `visibility_id`)
+VALUES
+(@option_group_id_arel, {localize}'{ts escape="sql"}Deferred Revenue Account is{/ts}'{/localize}, @option_group_id_arel_val+1, 'Deferred Revenue Account is', NULL, 0, 0, @option_group_id_arel_wt+1, {localize}'Deferred Revenue Account is'{/localize}, 0, 1, 1, 2, NULL);
```

Comment:
If you are going to do +1 on arel_val and arel_wt then please do it on arel itself as well.